manifests/init.pp

```diff
@@ -49,6 +49,9 @@
   Hash $jira_config_properties                                      = {},
   Boolean $datacenter                                               = false,
   $shared_homedir                                                   = undef,
+  $ehcache_listener_host                                            = undef,
```

Comment:
can you please add datatypes for the new parameters?

Reply:
We can probably use this type? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp
In this case please update the stdlib dependency in metadata.json to at least 4.25.0

Reply:
Added data types and bumped stdlib dependency